tests/server.py

```diff
@@ -739,9 +742,24 @@ def setup_test_homeserver(
             },
         }
     else:
+        if SQLITE_PERSIST_DB:
+            test_db_location = os.path.abspath("test.db")
```

Comment:
I think this needs something saying it gets into `_trial_temp`?

```suggestion
            # The current working directory is in _trial_temp, so this gets created within that directory.
            test_db_location = os.path.abspath("test.db")
```